apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/internal/AJ.kt

```diff
@@ -0,0 +1,45 @@
+package com.apollographql.apollo.cache.normalized.internal
+
+import com.apollographql.apollo.api.ResponseField
+
+/**
+ * A hierarchy of Json object with their fields attached used for normalization
+ */
+sealed class AJElement
```

Comment:
I would recommend to change the name to smth different, as `AJ` is so abstract and not very descriptive for `augmented json`. Couple suggestion as this only related to cache normalization maybe `NormalizationNode/ Element`, `CacheNormalizationNode / Element`?

Reply:
Renamed to `NormalizationIR`